packages/xstate-wallet/src/chain.ts

```diff
@@ -28,6 +28,9 @@ export interface Chain {
   getChainInfo: (channelId: string) => Promise<ChannelChainInfo>;
   chainUpdatedFeed: (channelId: string) => Observable<ChannelChainInfo>;
   deposit: (channelId: string, expectedHeld: string, amount: string) => Promise<void>;
+  ethereumEnable: () => Promise<void>;
```

Comment:
Is there a different place than the `Chain` interface this could live? e.g., this is not relevant when run inside the `hub`, but everything else is relevant. Perhaps a `Web3Chain` extension of `Chain`?